pushapkscript/docker.d/worker.yml

```diff
@@ -14,33 +14,6 @@ taskcluster_scope_prefixes:
 products:
   $flatten:
     $match:
-      'COT_PRODUCT == "firefox" && ENV == "prod"':
-        - product_names: [ "beta", "aurora", "release" ]
-          digest_algorithm: SHA1
-          override_channel_model: 'choose_google_app_with_scope'
-          apps:
-            release:
-              default_track: production
-              service_account: { "$eval": "GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_RELEASE" }
```

Comment:
The fennec nightly account can be deleted!

The Google Play accounts for `beta` and `release` are still used by the `mobile` workers. I think we should keep them around.

Reply:
Yeah, I meant the nightly ones, sorry for confusion. Since we don't have dedicated workers for just nightly, we can only remove SOPS credentials and the exporting of these env vars for the nightly ones, in this cleanup.